package.json

```diff
@@ -33,11 +33,14 @@
     "lint": "./node_modules/.bin/eslint src/"
   },
   "devDependencies": {
+    "babel-eslint": "^9.0.0",
     "eslint": "^5.4.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.1"
+    "eslint-plugin-react": "^7.11.1",
+    "react-test-renderer": "^16.4.2",
```

Comment:
I know we're not using *renderer* but It's still here 😕

Reply:
I have done away with it